### DISCUSSION TITLE
anyone know have info about a new unerollment exploit

### DISCUSSION THREAD

**Discussion by nathblade16** [3kh0/ext-remover#1149] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
my school is going through this mass chromebook unenrollment thing, someone even came up to me asking if they had a usb that they could use, I didn't have one but I still asked questions, ended up finding out they where trying to unenroll probably not knowing what they were doing, anyways altough I have a 0% chance of using it as I use a personal laptop for school and don't like unenrollment still want info on how this thing works?

**Comment by AshtonDavies** [comment, marked as answer]
There are new and upcoming unenrollment methods in [Titanium Network](https://discord.gg/unblock) if you're interested in checking them out.

**Reply by S-PScripts** [reply]
i got banned there for ban evasion 😭  i didn't do nothin

**Reply by nathblade16** [reply]
> i got banned there for ban evasion 😭 i didn't do nothin

I just don't want to be on that server, that simple, anyways the only reason why I want more info about this is I want to know how it works and possibly make a tool that can save chromebooks that would otherwise be e waste because the previous owners forgor to unenroll so basically help people who got scammed without the scammers even intending to scam them

**Reply by AshtonDavies** [reply]
> i got banned there for ban evasion 😭  i didn't do nothin 

A sneaky anonymous silently removed my programmer role for no valid reason one day, so I understand if you think the staff team is ridiculous, because it is. Still, none of them have admitted to that crime, but I thought it would be glorious to expose it to its truth so we could fix that silly bastard.

The support team is also terrible and should be buffed.

Riftriot sits at his desk for hours just saying "Nah, lol" to kids and has no purpose in life.

So now you understand why we have spinoff servers, but the latest stuff doesn't get to other places very fast (sorry).

**Reply by S-PScripts** [reply]
> > i got banned there for ban evasion 😭  i didn't do nothin
> 
> A sneaky anonymous silently removed my programmer role for no valid reason one day, so I understand if you think the staff team is ridiculous, because it is. Still, none of them have admitted to that crime, but I thought it would be glorious to expose it to its truth so we could fix that silly bastard.
> 
> The support team is also terrible and should be buffed.
> 
> Riftriot sits at his desk for hours just saying "Nah, lol" to kids and has no purpose in life.
> 
> So now you understand why we have spinoff servers, but the latest stuff doesn't get to other places very fast (sorry).

ok maybe i did do something...

 i used to link leak but then someone i forgot who said for me to stop. even though i had stopped, someone else a few months later in December told some staff in the tn discord that i link leaked so i got banned. i joined on an alt 6 months later and i created an incognito post and also posted a few webviews but once i decided to reveal who i was i was banned instantly.


**and by the way, the proxies i link leaked came from ghostedzoomer's list.**
